pyiron_lammps/compatibility/file.py

```
@@ -222,3 +227,30 @@ def _modify_input_dict(
         return lmp_tmp_lst
     else:
         return lmp_str_lst
+
+
+def _get_potential(potential):
+    if isinstance(potential, str):
+        potential_dataframe = get_potential_by_name(
+            potential_name=potential, resource_path=resource_path
```

Ruff (0.14.6)

Undefined name `resource_path`

(F821)